test/service.ts

```diff
@@ -676,8 +700,8 @@ describe('GrpcService', () => {
         };
 
         grpcService.request(PROTO_OPTS, REQ_OPTS, assert.ifError);
-
-        const request = retryRequestOptions.request();
+        // tslint:disable-next-line no-any
+        const request = (retryRequestOptions as any).request();
```

Comment:
Hm this one smells funky - we can't cast as `retryRequest.Options`?

Reply:
I tried but, its looking for some arguments, and i really don't have idea its good to pass, or what to pass! here also typescript is upset, suggestions welcome :)

Reply:
Yikes, yeah thought that'd be the case. Did you try casting as `Function`? It's just a bit stricter than `any`

Reply:
I tried your suggestion, but getting error!